fiat-go/32/poly1305/poly1305.go

```diff
@@ -30,6 +30,16 @@ package poly1305
 type uint1 uint8
 type int1 int8
 
+// The type LooseFieldElement is a field element with loose bounds.
```

Comment:
```suggestion
// LooseFieldElement is a field element with loose bounds.
```

Go style omits prefixes like "The function" and "The type".

Reply:
Done